schemas/com.vimeo/cue_point_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo cue point event, fired when a cue point is reached in a video.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "cue_point_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "description": "The ID of the cue point."
+    },
+    "time": {
```

Comment:
I'm not an expert here... but is `time` an awkward column name in the warehouse?  I vaguely remember it needs escaping when writing ad-hoc queries, but I might be wrong.  It won't break anything, so you don't have to change it.